pkg/diff/diff.go

```diff
@@ -51,21 +51,23 @@ func hashEmptyDir() []byte {
 }
 
 type Entry struct {
-	path string
-	mode fs.FileMode
-	hash []byte
-	err  error
+	path    string
+	mode    fs.FileMode
+	modTime int64
+	hash    []byte
+	err     error
 }
 
 func (e *Entry) toPb() *pb.Entry {
 	return &pb.Entry{
-		Path: e.path,
-		Mode: uint32(e.mode),
-		Hash: e.hash,
+		Path:    e.path,
+		Mode:    uint32(e.mode),
+		ModTime: e.modTime,
```

Comment:
We don't actually write out these pbs anywhere that is stored for a long time right? Like they are only ephemeral such that this change won't cause any forwards/backwards compatibility issues with already stored summaries? I guess since it is a field add it would also be ok if we did right?

Reply:
Yeah they are only temporary, but after reviewing the PR we only need these temporarily in the `Entry` struct but could discard them before writing to disk.

Reply:
That's right, all of the `fsdiff` files are super temporary in nature. If this was rolled out into production we could be a bit more careful about how we handle the new field, but there is nowhere that this will exist on disk for a long time.